absl/synchronization/internal/win32_waiter.h

```diff
@@ -17,6 +17,7 @@
 #define ABSL_SYNCHRONIZATION_INTERNAL_WIN32_WAITER_H_
 
 #ifdef _WIN32
+#include <windows.h>
```

Comment:
We cannot `#include <windows.h>` in headers because it's non-modular and hijacks global identifiers like `ERROR` and `StrCat`.